src/System.Net.Requests/src/project.json

```diff
@@ -4,6 +4,7 @@
       "dependencies": {
         "Microsoft.NETCore.Platforms": "1.0.1",
         "System.Collections": "4.4.0-beta-24619-02",
+        "Microsoft.Win32.Primitives": "4.4.0-beta-24619-02",
```

Comment:
LOL.. yes it was originally and I was doing some testing to figure out how we got into this state and it looks like I pasted it back on the wrong line.  I will update.